numba/_math_c99.c

```diff
@@ -23,7 +23,7 @@
 static const double ln2 = 6.93147180559945286227E-01;
 static const double two_pow_m28 = 3.7252902984619141E-09; /* 2**-28 */
 static const double two_pow_p28 = 268435456.0; /* 2**28 */
-static const double zero = 0.0;
+// static const double zero = 0.0; // UNUSED
```

Comment:
`zero` does get used on line 147 if `Py_NAN` is not defined. Could surround this in the same `ifdef`?

Reply:
Ah yes, thanks. Whilst I don't think we'd hit this as all the supported platforms support NaN, it's probably best to future proof it!